apps/opendosm/dashboards/exchange-rates/index.tsx

```diff
@@ -68,7 +69,13 @@ const ExchangeRatesDashboard: FunctionComponent<ExchangeRatesDashboardProps> = (
         header={[t("header"), "dark:text-white"]}
         description={[t("description"), "dark:text-white"]}
         last_updated={last_updated}
-        agencyBadge={<AgencyBadge agency="mampu" />}
+        agencyBadge={
+          <AgencyBadge
+            name={t("agencies:bipd.full")}
```

Comment:
I think @leneherng ady moved all division/dept level i18n to its own namespace.

So, `t("division:bipd.full")`. Add the division namespace to autoload in next-i18n config file